Source/DataSources/KmlDataSource.js

```diff
@@ -1474,27 +1628,370 @@ define([
         console.log('KML - Unsupported feature: ' + node.localName);
     }
 
+    var RefreshMode = {
+        INTERVAL : 0,
+        EXPIRE : 1,
+        STOP : 2
+    };
+
+    var scratchCartesian2_1 = new Cartesian2();
+    var scratchCartesian3_1 = new Cartesian3();
+    var scratchCartesian3_2 = new Cartesian3();
+    var scratchCartesian3_3 = new Cartesian3();
+    var scratchCartesian3_4 = new Cartesian3();
+    var scratchCartographic = new Cartographic();
+    var horizonPoints = [new Cartesian3(), new Cartesian3(), new Cartesian3(), new Cartesian3()];
+
+    function computeHorizonQuad(camera) {
```

Comment:
I agree with @pjcozzi let's add the required function to the camera (and add tests).

@bagnell, should `computeHorizonQuad` and the code from `DepthPlane.js` get refactored into a shared private function?